.circleci/config.yml

```diff
@@ -51,6 +51,8 @@ jobs:
           keys:
             - serf-modcache-v1-{{ checksum "go.mod" }}
 
+      - run: sudo apt-get update && sudo apt-get install -y rsyslog
```

Comment:
Lifted this from consul's circleci configs to get a syslog-related serf test to actually pass.